### DISCUSSION TITLE
paddleocr pip安装的库 仅支持DB

### DISCUSSION THREAD

**Discussion by gengyanlei** [PaddlePaddle/PaddleOCR#12707] [Q&A] [answered]
你好，paddleocr pip安装的库 仅支持DB，不支持其它检测网络，我点击查看pip安装的代码，里面明明有其它检测网络参数，但是却无法使用。
这样好鸡肋啊，不能指定参数，那推广这个pip库有啥意义呢，只能使用1个固定的网络，不能传参指定。
那readme中快速开始就没有意义了，让人通过pip安装，却无法使用传参指定其它网络。
哎，折腾半天。。。

**Comment by Sunting78** [comment, marked as answer]
同学您好，也可以自定义模型请参考：https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/whl.md#3-%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A8%A1%E5%9E%8B

**Comment by gengyanlei** [comment]
@Sunting78 
你好，知道了，就是觉得whl都安装好了，为啥不能传参修改一些训练好的呢。这个不重要了，我下载仓库了。
目前，我有个疑问，PPOCRLabel四边形标注必须要求矩形么？还是训练时，代码处理这种四边形为最小外接矩形？我看readme里面要求4边形，但是没说必须为矩形。按照DBNet的理论，应该任意形状都可以吧？？但是我现在就想直接用ppocr，那么它的输入要求就是普通四边形，里面是不是有代码把四边形转成了最小外接矩形呢？？？
谢谢了

**Reply by SWHL** [reply]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
